Document/PostManager.php

```diff
@@ -39,6 +43,55 @@ public function findOneBySlug($year, $month, $day, $slug)
     }
 
     /**
+     * @param string        $permalink
+     * @param BlogInterface $blog
+     *
+     * @return PostInterface
+     */
+    public function findOneByPermalink($permalink, BlogInterface $blog)
```

Comment:
> Someone (me) could extend it and add other permalink fields (e.g. a category).

Do you have a code sample?

`findOneByPermalink` has quite much logic inside, maybe some parts could be extracted.